cargo-apk/README.md

```diff
@@ -159,6 +159,11 @@ orientation = "landscape"
 # Unset by default, or "true" when targeting Android >= 31 (S and up).
 exported = "true"
 
+# See https://developer.android.com/guide/topics/manifest/activity-element#resizeableActivity
+#
+# Unset by default, or "true" when targeting Android >= 24 (Nougat and up).
```

Comment:
Not entirely true, `cargo-apk` doesn't set this value at all. The matching comment on `exported` above exists specifically to document this logic on the `cargo-apk` side:

https://github.com/rust-windowing/android-ndk-rs/blob/93558ca7b03f1818bb2bfc832f9b2bf52742b557/cargo-apk/src/apk.rs#L83-L88

Instead, what you described here is how Android _treats_ this field by default on Android >= 24.

Reply:
Ah, I see, that's true. I'll change the comment.